src/math/unary_ops_test.ts

```diff
@@ -488,6 +558,47 @@ import {Array1D, Array2D, Scalar} from './ndarray';
       const r = math.exp(a);
       test_util.expectArraysClose(r, [Math.exp(1), NaN, 1]);
     });
+
+    it('gradients: Scalar', math => {
+      const a = Scalar.new(5);
+      const dy = Scalar.new(3);
+
+      const gradients = math.vjp(() => math.exp(a), a, dy);
+
+      expect(gradients.shape).toEqual(a.shape);
+      expect(gradients.dtype).toEqual('float32');
+      test_util.expectArraysClose(gradients, [3 * Math.exp(5)], 1);
```

Comment:
@nsthorat Here gradient value and expected value have a difference of more than `0.1`. Not sure why.